external_drivers/minimal_example/mdalDummyExternalDriver.cpp

```diff
@@ -13,7 +13,7 @@
 
 #include "mdal_external_driver.h"
 
-#include "limits.h"
+#include <limits>
```

Comment:
only a form suggestion, can you just move this #include to line 13? to have std include together.